app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntrySheet.kt

```diff
@@ -383,6 +383,16 @@ class BloodSugarEntrySheet : BottomSheetActivity(), BloodSugarEntryUi, RemoveBlo
     bloodSugarReadingUnitButton.text = getString(R.string.bloodsugarentry_mg_dl)
   }
 
+  override fun showBloodSugarUnitPreferenceButton() {
+    bloodSugarReadingUnitButton.visibility = View.VISIBLE
+    bloodSugarReadingUnitLabel.visibility = View.GONE
+  }
+
+  override fun hideBloodSugarUnitPreferenceButton() {
+    bloodSugarReadingUnitButton.visibility = View.GONE
+    bloodSugarReadingUnitLabel.visibility = View.VISIBLE
```

Comment:
Same comment as above, can we move the showing on `bloodSugarReadingUnitLabel` into its own method.